qcfractal/cli/qcfractal_manager.py

```diff
@@ -205,7 +207,13 @@ def main(args=None):
         client = None
     else:
         # Connect to a specified fractal server
-        print(settings.server.fractal_uri, settings.server.dict(skip_defaults=True, exclude={"fractal_uri"}))
+        print_string = "Attempting to connect to FractalClient at: {}".format(settings.server.fractal_uri)
+        other_data = settings.server.dict(skip_defaults=True, exclude={"fractal_uri"})
+        if "password" in other_data:
+            other_data["password"] = "{provided}"
+        if other_data:
+            print_string += "\n   With credentials: {}".format(other_data)
```

Comment:
This is printed internally within the Manager. I don't think we want to duplicate this here.

Reply:
Okay, there was an outstanding print statement, I think from debugging now that I think about it, but it was showing the password too. I'll remove it

Reply:
Just we shouldn't use print. Just logging and all of that is initialize in the manager itself.